requirements/requirements.in

```diff
@@ -1,6 +1,5 @@
 dictdiffer~=0.8
 dataclasses-json~=0.3
-gunicorn~=22.0
```

Comment:
I would keep this dependency. I know this is a transitional dependency through mlflow, but mlflow is extras_require. A user can install gordo without it